libraries/provider_component_runit_supervisor_systemd.rb

```diff
@@ -23,6 +23,15 @@ class Systemd < Chef::Provider::LWRPBase
             source "runsvdir-start.service.erb"
           end
 
+          # This cookbook originally installed its unit files in /usr/lib/systemd/system.
+          execute "echo" do
```

Comment:
Nitpicking, but why not give the resource a name, e.g. `execute "cleanup old unit files" do ...`? The `echo` seems a bit odd, doesn't it? (Maybe some convention I know nothing about?)